### PR TITLE
Task nxp 27712 make pipeline great again

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

**Review by kevinleturc — DISMISSED** [review]
Good work 👌

**Review by kevinleturc — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -16,87 +16,316 @@
...
+    MONGODB_CHART_NAME = 'mongodb'
+    MONGODB_CHART_VERSION = '7.14.2'
+    ELASTICSEARCH_CHART_NAME = 'elasticsearch'
+    ELASTICSEARCH_CHART_VERSION = '7.9.2'
+    KAFKA_CHART_NAME = 'kafka'
+    KAFKA_CHART_VERSION = '11.8.8'
+    NUXEO_CHART_NAME = 'nuxeo'
+    NUXEO_CHART_VERSION = '~2.0.0'
+    NUXEO_DOCKER_REPOSITORY = "${isTriggeredByNuxeoPR() ? DOCKER_REGISTRY : PUBLIC_DOCKER_REGISTRY}/nuxeo/nuxeo"
+    NUXEO_VERSION = "${params.NUXEO_VERSION?.trim() ? params.NUXEO_VERSION : '11.x'}"
```
Comment (troger) [inline Jenkinsfile:172]: Why not using the parameter default value?

**Review by troger — APPROVED** [review]
👍 👍

File: Jenkinsfile (hunk 1 above)
Comment (ataillefer) [reply Jenkinsfile:172]: To detect if the trigger was not form a PR with `isTriggeredByNuxeoPR`, in which case the param should be empty.

**Pull request merged**